packages/cli/src/index.ts

```diff
@@ -12,11 +11,6 @@ export class CLI {
   }
 
   public run() {
-    this.cmd
-      .command("install")
-      .description("installs AppSignal")
-      .action(install)
```

Comment:
If this command definition is removed, calling `npx @appsignal/cli install` will do nothing, as the command will not be defined, and the function in `install.ts` (which emits the console log message explaining to the customer that the installer is deprecated) will never be called.